src/test/java/TestBasicAnnotation.java

```diff
@@ -0,0 +1,104 @@
+
+import classfortest.Person;
+import classfortest.StudentBreskulTeam;
+import org.breskul.connectivity.annotation.Column;
+import org.breskul.connectivity.annotation.Entity;
+import org.breskul.connectivity.annotation.Id;
+import org.breskul.connectivity.annotation.Table;
+import org.junit.Test;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.Objects;
+
+public class TestBasicAnnotation {
+
+    @Test
+    public void existAnnotationTable() {
+
+        final var annotationEntityIsPresent = Person.class.isAnnotationPresent(Entity.class);
```

Comment:
Instead of using reflection API I would suggest building the tests based on the session method: 
public <T> T find(Class<T> entityType, Object id)

So you can create several tables and records in DB and try to find them for correspondent Class<T>.

Reply:
will be written in another task when the find method is found